src/bun_cpp_utils.cpp

```diff
@@ -5,23 +5,30 @@
 
 #include <cxxabi.h>
 
+template<typename T>
+struct deleter
+{
+	void operator() (T data) {
```

Comment:
This should be a const method, and we want to make sure `data` is a pointer, so that's `void operator()(T* data) const`

Reply:
We could also move the template to the operator, `template <typename T> void operator()(T *data) const`, and not have to repeat the deleted type in `unique_ptr`.